index.html

```diff
@@ -1,380 +1,293 @@
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="styles.css" />
+  <link rel="stylesheet" type="text/css" href="styles1.css" media="screen and (min-width:992px)" />
+  <link rel="stylesheet" type="text/css" href="styles2.css"
+    media="screen and (min-width:400px) and (max-width:991px)" />
+  <script src="https://kit.fontawesome.com/eb9ab0d66a.js" crossorigin="anonymous"></script>
+  <title>Portfolio's Main Page</title>
+</head>
+
+<body>
+  <header>
+    <h2 class="logo">My logo</h2>
+    <img src="images/Union.png" class="ham" alt="hamburger" />
+    <ul>
+      <li>Portfolio</li>
+      <li>About</li>
+      <li>Contact</li>
+    </ul>
+
+    <nav class="mobile-menu none">
+      <button type="button" class="mobile-menu-btn">
+        <img src="images/cross.svg" alt="hamburger" />
+      </button>
       <ul>
-        <li>Portfolio</li>
-        <li>About</li>
-        <li>Contact</li>
+        <li>
+          <a href="#portfolio">Portfolio</a>
+        </li>
+        <li>
+          <a href="#about">About</a>
+        </li>
+        <li>
+          <a href="#contact">Contact</a>
+        </li>
       </ul>
+    </nav>
+  </header>
 
-      <nav class="mobile-menu none">
-        <button type="button" class="mobile-menu-btn">
-          <img src="images/cross.svg" alt="hamburger" />
-        </button>
-        <ul>
-          <li>
-            <a href="#portfolio">Portfolio</a>
-          </li>
-          <li>
-            <a href="#about">About</a>
-          </li>
-          <li>
-            <a href="#contact">Contact</a>
-          </li>
-        </ul>
-      </nav>
-    </header>
+  <section class="header-section">
+    <h1>I'm Nabil<br />Glad to see you!</h1>
+    <p>
+      I'm a software developer! I can help you build a product , feature or
+      website Look through some of my work and experience! If you like what
+      you see and have a project you need coded, don't hestiate to contact me.
+    </p>
+    <span class="connect">LET'S CONNECT</span>
+    <ul class="social">
+      <li>
+        <a href="https://twitter.com/NeoNabil2"><img class="icons" src="images/twitter.png"
+            alt="Access to Twitter Account" /></a>
+      </li>
+      <li>
+        <a href="https://www.linkedin.com/"><img class="icons" src="images/Linkedinicon.png"
+            alt="Access to LinkedIn Account" /></a>
+      </li>
+      <li>
+        <a href="https://medium.com/"><img class="icons" src="images/medium.png" alt="Access to Meduim Account" /></a>
+      </li>
+      <li>
+        <a href="https://github.com/NabilHY"><img class="icons" src="images/github.png"
+            alt="Access to GitHub Account" /></a>
+      </li>
+      <li>
+        <a href="https://angel.co/"><img class="icons" src="images/angellist.png"
+            alt="Access to Angellist Account" /></a>
+      </li>
+    </ul>
+  </section>
+
+  <section id="portfolio" class="works">
```

Comment:
- [ ] Also make sure the projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript.** Check an example of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95

Reply:
- Thank you for reviewing my code, I made the changes that you suggested, which include : 
- Making the work section gets created dynamically through JS implementation.
- I edited the styling as much as I could to match the one on the Figma template.